docs/layers/hexagon-layer.md

```diff
@@ -131,15 +131,15 @@ Whether to enable cell elevation. Cell elevation scale by count of points in eac
 
 - Default: `100`
 
-Filter bins and re-calculate color by `upperPercentile`. Hexagons with counts
-bigger than the upperPercentile counts will be hidden.
+Filter bins and re-calculate color by `upperPercentile`. Hexagons with value
+larger than the upperPercentile will be hidden.
```

Comment:
This reason of doing this is to filter out really big numbers and re calculate color. When colors for are recalculated, the larger bins that fall outside the percentile range will not have accurate color any more, so we have to hide it. Also if we want to display transparence, we have to always enable depthTest